src/OpenSage.Game/Content/AptLoader.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenSage.Content
+{
+    class AptLoader
```

Comment:
Will this be filled in later? Should it be removed for now?

Reply:
Will be filled